single_tech_samples/databricks/sample4_ci_cd/devops/templates/jobs/deploy-libs-job.yml

```diff
@@ -0,0 +1,62 @@
+parameters:
+- name: environmentName
+  type: string
+- name: databricksDomain
+  type: string
+- name: databricksToken
+  type: string
+- name: databricksNotebookPath
```

Comment:
Perhaps it makes more sense to call this `databricksLibsPath` ?

Reply:
@devlace Yes! It's been changed to databricksLibsPath.